docs/custom-flows/email-password.mdx

```diff
@@ -366,9 +365,9 @@ This guide will walk you through how to build a custom email/password sign-up an
     </Tab>
 
     <Tab>
-      Create a route group with a layout that redirects users if they're already signed in, as shown in the following example:
+      Create an `(auth)` route group. This will group your sign-up and sign-in pages. In the `(auth)` group, create a `_layout.tsx` file with the following code. The [`useAuth()`](/docs/references/react/use-auth) hook is used to access the user's authentication state. If the user is already signed in, they will be redirected to the home page.
```

Comment:
```suggestion
      1. Create an `(auth)` route group. This will group your sign-up and sign-in pages. 
      1. In the `(auth)` group, create a `_layout.tsx` file with the following code. The [`useAuth()`](/docs/references/react/use-auth) hook is used to access the user's authentication state. If the user is already signed in, they will be redirected to the home page.
```